skype-sender-exercise/test_send_msg_testchannel.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import os, json
+from skype_config import channel, FirstName
+
+def test_send_msg_testchannel():
+    #send a message to skype channel "Test Qxf2Bot"
+    API_KEY = os.environ.get('API_KEY')
+    #FirstName = os.environ.get('FirstName')
+    
+    headers = {'Content-Type': 'application/json',}
+	
+    msg = "Hi, This is a message from %s" %(FirstName)    
```

Comment:
Since first name is very much tied to this test case, do you think we should keep it configurable? We can actually do it if we see more tests using first name.

Code looks good.

Reply:
I did that as Arun mentioned it as $your_firstname"